girder/utility/plugin_utilities.py

```diff
@@ -88,6 +88,22 @@ def loadPlugins(plugins, root, appconf, apiRoot=None, curConfig=None):
     return root, appconf, apiRoot
 
 
+def getPluginParentDir(name, curConfig=None):
+    """
+    Finds the directory a plugin lives in and returns it. This throws
+    an exception if it can't find a directory named name in any of the
+    set plugin_directory paths.
+
+    :params name: The name of the plugin (i.e. its directory name)
```

Comment:
That's right. There is the human readable name from the plugin.yml file, but the authoritative name/id of the plugin is its directory name.